Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:20.04
+WORKDIR /app
+COPY package.json .
```

Comment:
It looks like lock file copy is missing, is it correct? As far as I can see we can copy 'yarn.lock' file inside container to use it during dependencies installation.